indigo_app/js/indigo.js

```diff
@@ -11,6 +11,7 @@ class IndigoApp {
 
     this.createVueComponents(document);
     window.dispatchEvent(new Event('indigo.components-created'));
+    window.globalVue = Vue;
```

Comment:
An instance of the `IndigoApp` class is created in https://github.com/laws-africa/indigo/blob/0e4ee1dc9dbd7a63e5e4365c1138ce9691089263/indigo_app/js/main.js and stored in window.indigoApp, so you can use `window.indigoApp.Vue`, rather than adding more global variables.

Reply:
done